examples/plot_localizer_analysis.py

```diff
@@ -82,28 +83,32 @@
 
 # simplest ones
 contrasts = {}
-n_columns = len(design_matrix.names)
+n_columns = len(column_names)
 for i in range(n_conditions):
-    contrasts['%s' % design_matrix.names[2 * i]] = np.eye(n_columns)[2 * i]
+    contrasts['%s' % column_names[2 * i]] = np.eye(n_columns)[2 * i]
```

Comment:
You can take the matrix creation out of the loop and include the '2 * i' in the range object. Also the '%s' is not really useful any more:

```python
identity = np.eye(n_columns)

for even_index in range(0, n_columns, 2):
    contrasts[column_names[even_index]] = identity[even_index]
```